Source/WebKit/UIProcess/API/Cocoa/WKWebViewTesting.mm

```diff
@@ -221,8 +222,13 @@ - (void)_requestActiveNowPlayingSessionInfo:(void(^)(BOOL, BOOL, NSString*, doub
         return;
     }
 
-    _page->requestActiveNowPlayingSessionInfo([handler = makeBlockPtr(callback)] (bool active, bool registeredAsNowPlayingApplication, String title, double duration, double elapsedTime, uint64_t uniqueIdentifier) {
-        handler(active, registeredAsNowPlayingApplication, title, duration, elapsedTime, uniqueIdentifier);
+    _page->requestActiveNowPlayingSessionInfo([handler = makeBlockPtr(callback)] (bool registeredAsNowPlayingApplication, WebCore::NowPlayingInfo&& nowPlayingInfo) {
+        BOOL isActive = nowPlayingInfo.allowsNowPlayingControlsVisibility;
+        NSString *title = nowPlayingInfo.title;
```

Comment:
nowPlayingInfo.title is a `String`, not a NSString

why create temporary variable, just pass the values from the structure directly to the completion handler.